examples/sample-input-byo.tfvars

```diff
@@ -6,7 +6,6 @@
 # These required variables' values MUST be provided by the User
 prefix         = "<prefix-value>"
 location       = "<azure-location-value>" # e.g., "eastus2"
-ssh_public_key = "~/.ssh/id_rsa.pub"
```

Comment:
@enderm if you can add the `ssh_public_key` entries back into the sample files. As we chatted a bit on Friday, I believe the variable `ssh_public_key` needs to be present in order for people to see it and know what to change. Especially since it controls access to their VM's. Might need to move the variables location like you have in the doc.